contents/handbook/company/values.md

```diff
@@ -16,19 +16,21 @@ This enables the strongest community growth possible. It causes the core team to
 
 ## We haven't built our defining feature yet
 
-We will never stop iterating.
+Ideas are fragile. Be direct but encouraging when you encounter new ideas. If we default to providing negative criticism of new ideas, then they'll simply stop coming. Instead, default to "yes and" rather than "no but", before rejecting an idea.
 
-You learn faster and help PostHog perform better by getting what you're working on into the real world. We expect you to ship new designs, features, or whatever is needed for your role in tiny chunks, frequently, and often before you feel ready.
+We should never stop iterating. You learn faster and help PostHog perform better by getting what you're working on into the real world. We expect you to ship new designs, features, or whatever is needed for your role in tiny chunks, frequently, and often before you feel ready.
 
 ## Everyone codes
 
 ...although this doesn't mean everyone has to be a software developer, and not everyone needs experience in this before they join. Our platform's adoption starts with developers using our open source project so we use GitHub to share most of our work publicly and to build a large community of technical users.
 
 No matter your role, being able to use the basics of GitHub helps you understand our audience. Beyond that, we'll encourage you to build your technical skill, rather than delegating more challenging tasks to others, so you become a more effective contributor.
 
-## Step on toes
+## Trust and feedback over process
 
-We have ambitious goals. This means people need to pick up what they know is most important at any time, not what they feel their job title means they are allowed to do.
+There are two ways to scale - trust and feedback, or process. We choose the former because we're building a wide platform with many products, so autonomy is more important than control. We hire people that work well with high level direction and are will step on toes if needed to get things done.
```

Comment:
```suggestion
There are two ways to scale - trust and feedback, or process. We choose the former because we're building a wide platform with many products, so autonomy is more important than control. We hire people that work well with high level direction and will step on toes if needed to get things done.
```